srcpkgs/python3-tzdata/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-tzdata'
+pkgname=python3-tzdata
+version=2022.2
+revision=1
+wrksrc="tzdata-${version}"
+build_style=python3-pep517
+hostmakedepends="python3-wheel"
+depends="python3"
+checkdepends="python3-pytest-subtests python3-pytest-xdist"
+short_desc="Python package wrapping the IANA time zone database"
+maintainer="icp <pangolin@vivaldi.net>"
+license="Apache-2.0"
+homepage="https://github.com/python/tzdata"
+changelog="https://raw.githubusercontent.com/python/tzdata/master/NEWS.md"
+distfiles="https://github.com/python/tzdata/archive/refs/tags/${version}.tar.gz"
+checksum=7683412157d401fd9f2e1bb945a4f2e3971542d49623756ec66dcf09ec1f955e
+make_check_pre="env PYTHONPATH=src"
```

Comment:
> This is a Python package containing zic-compiled binaries for the IANA time zone database. It is intended to be a fallback for systems that do not have system time zone data installed (or don't have it installed in a standard location), as a part of [PEP 615](https://www.python.org/dev/peps/pep-0615/)

IOW, No to this package

Reply:
Can I use the approach demonstrated in the `python3-pytz` [template](https://github.com/void-linux/void-packages/blob/master/srcpkgs/python3-pytz/template#L17-L22)? I tested locally and it seems to work. Otherwise I can't figure out how to satisfy the dependency for `python3-pytz_deprecation_shim`.

Reply:
Just remove it.

Reply:
The requirement should be something like this: https://github.com/regebro/tzlocal/blob/c2b71e9431330a81591e10c30c4b896540e634c4/setup.cfg#L33 instead of https://github.com/pganssle/pytz-deprecation-shim/blob/bf0174adfce698e280c623d17fe8e82961de0c48/setup.cfg#L36

Reply:
Okay, now I understand - thanks!

Reply:
You need to patch it out.